### PR TITLE
Initial commit for Italian texture localization

### PR DESCRIPTION
**Pull request by Macapple96**
Contributing italian texture localization to the project!

- Added `it` locale;
- Added `it/pdn/` folder which contains paintdotnet source image files for easier editing;
-- Includes manually "cleaned up" blank jokers as layers, separated from text; 
- Added overrides for italian texture generation in `consumables.json`;
- Generated italian spritesheets in `dist/it/`;

No mod uses these files yet; Playtested with manual file substitution method.
Due to italian grammar, some decorative elements in booster packs (such as _Jumbo Arcana packs_) may look a bit off.

### CODE REVIEW

**Comment by Signez** [comment]
So cool! I'll take a few days to look into it (especially the Paint.net files, I'll have to boot up my Windows PC to check them out), but it's very cool to see another contribution to this pipeline with another language! Thanks a lot.

File: locales/it/jokers/j_todo_list.png
Comment (Signez) [inline locales/it/jokers/j_todo_list.png:1]: I am afraid it's impossible to read “JOKER” by reading each of the first letters of the todolist in the Italian translation (it reads "JPKER"), whereas it's possible in every other translation we already merged and in the original asset.

|Original|French|Romanian|Italian|
|--|--|--|--|
|Original|French|Romanian|Italian|

If you feel it's too hard to fix / you don't care, it's fine, feel free to tell me and I can merge it as-is.

**Review by Signez — CHANGES_REQUESTED** [review]
Great work! I mainly have one remark that could improve that translation before a merge.

File: locales/it/jokers/j_todo_list.png
Comment (Macapple96) [reply locales/it/jokers/j_todo_list.png:1]: Hello! I just updated the sprite as requested. 

I'm a bit embarassed to admit the reason it didn't read 'JOKER' from the beginning was that I _never_ noticed in the first place 😆 

Here's the new version!

| |todo_list.png|reserved_parking.png|
| :--- | :---: | :---: |
| *New* | j_todo_list | j_reserved_parking
| *Old* | tmpF7B6 tmp | tmp2464 tmp |

**Review by Signez — APPROVED** [review]

**Comment by Signez** [comment]
Thanks again for your contributions 🎉

**Pull request merged**